### ISSUE TITLE
Bad UX/translations for repeating events

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/calendar#103]
The problem is, that the input field has no placeholder and therefor is quite confusing when the translation of the "Repeat every ..." is not well chosen by the translator.
![bildschirmfoto vom 2016-09-30 09-10-35](https://cloud.githubusercontent.com/assets/213943/18983579/08e4da3a-86ee-11e6-8e7d-f05c12437c2b.png)

E.g. german: "Wiederhole immer 3" just makes no sense. Maybe we can add a little placeholder, or even change the label text to "Repeat every ... week" (depending on week/day/month/...)

**Comment by georgehrke** [comment]
We are going to refactor the repeating evens dialog very soon.
see https://github.com/nextcloud/calendar/issues/10

**Comment by georgehrke** [comment]
> E.g. german: "Wiederhole immer 3" just makes no sense.

A better translation would definitely be "Wiederhole alle ..."

**Comment by nickvergessen** [comment]
I will do the translation change, once #92 is done...

**Comment by eppfel** [comment]
I think a huge improvement would be instant changes reflected in the calendar. You'd have visual feedback that helps you with this sometimes complex task. But this is something for another PR or even issue...

Any thoughts on feasibility?

**Comment by georgehrke** [comment]
> Any thoughts on feasibility?

The expansion of recurring events happens in the javascript of the calendar app, therefore this should definitely be possible. But I think if anything that's something we want for all properties:
- user changes title -> title of event changes 
- user selects another calendar in editor -> color of event changes
- user changes date and time -> event moves to new time and if necessary the visible day/week/month adapts
- ...

**Comment by eppfel** [comment]
@georgehrke Yes, direct bindings would be awesome. Let's make that into an issue then...

**Comment by georgehrke** [comment]
translation from »Wiederhole immer ... « to »Wiederhole alle« was fixed.
Everything else goes into #10
